src/components/06-accordion/accordion.njk

```diff
@@ -1,17 +1,17 @@
-<div class="usa-accordion{%
-  if accordion.variant %} usa-accordion--{{ accordion.variant }}{% endif %}"{%
-  if accordion.multiselectable == true %} aria-multiselectable="true"{% endif %}>
-  {% for item in accordion.items %}
-  <!-- Use the accurate heading level to maintain the document outline -->
-  <h2 class="usa-accordion__heading">
-    <button class="usa-accordion__button"
-      aria-expanded="{{ item.expanded | default('false') }}"
-      aria-controls="{{ accordion.id_prefix }}{{ item.id }}">
-      {{ item.title }}
-    </button>
-  </h2>
-  <div id="{{ accordion.id_prefix }}{{ item.id }}" class="usa-accordion__content usa-prose">
-    {{ item.content | safe }}
+{% for item in accordion.items %}
```

Comment:
Does this fix the Multiselectable in USWDS-Site? I don't think a loop is necessary here, unless we wanted to output them all on a single page. I'm using [alerts.config.yml](https://github.com/uswds/uswds/blob/bc7ca86a5de23227a60382ae44a52bd02e5e1b8e/src/components/05-alerts/alerts.config.yml) and [alerts.njk](https://github.com/uswds/uswds/blob/c6dc1044877bc532445e6297ab84266703f776f9/src/components/05-alerts/alerts.njk) for reference.

`{% for item in accordion.items %}` is for the individual accordion items. 
This is what I get for accordion variables with `{{ accordion | dump }}`
<details>
 <summary>{{ accordion | dump(2) }}</summary>

```js
{
  "items": [
    {
      "title": "First Amendment",
      "id": "a1",
      "expanded": true,
      "content": "<p>Congress shall make no law respecting an establishment of religion, or prohibiting the free exercise thereof; or abridging the freedom of speech, or of the press; or the right of the people peaceably to assemble, and to petition the Government for a redress of grievances.</p>\n"
    },
    {
      "title": "Second Amendment",
      "id": "a2",
      "content": "<p>A well regulated Militia, being necessary to the security of a free State, the right of the people to keep and bear Arms, shall not be infringed.</p> <ul><li>This is a list item</li><li>Another list item</li></ul>\n"
    },
    {
      "title": "Third Amendment",
      "id": "a3",
      "content": "<p>No Soldier shall, in time of peace be quartered in any house, without the consent of the Owner, nor in time of war, but in a manner to be prescribed by law.</p>\n"
    },
    {
      "title": "Fourth Amendment",
      "id": "a4",
      "content": "<p>The right of the people to be secure in their persons, houses, papers, and effects, against unreasonable searches and seizures, shall not be violated, and no Warrants shall issue, but upon probable cause, supported by Oath or affirmation, and particularly describing the place to be searched, and the persons or things to be seized.</p>\n"
    },
    {
      "title": "Fifth Amendment",
      "id": "a5",
      "content": "<p>No person shall be held to answer for a capital, or otherwise infamous crime, unless on a presentment or indictment of a Grand Jury, except in cases arising in the land or naval forces, or in the Militia, when in actual service in time of War or public danger; nor shall any person be subject for the same offence to be twice put in jeopardy of life or limb; nor shall be compelled in any criminal case to be a witness against himself, nor be deprived of life, liberty, or property, without due process of law; nor shall private property be taken for public use, without just compensation.</p>\n"
    }
  ],
  "id_prefix": "m-",
  "multiselectable": true
}
```
</details>

Reply:
Site doesn't need a fix for `multiselectable` since it doesn't document or demo this functionality yet. (Maybe it should.) This fixes an issue where all the accordions are wrapped in a single `usa-accordion` — look at the code on https://designsystem.digital.gov/components/accordion/ — it's wrapping all the accordions in a single `usa-accordion` where each one should get its own.

Reply:
nm — I guess the whole group _should_ be wrapped in a single `usa-accordion` — I'll revert

Reply:
reverted